bin/aws-kinesis-agent-babysit

```diff
@@ -8,7 +8,7 @@ PIDFILE=/var/run/$DAEMON_NAME.pid
 SERVICE="service $DAEMON_NAME"
 
 function get_agent_pid() {
-  echo $(ps --ppid $(cat $PIDFILE 2>/dev/null) ho pid 2>/dev/null)
+  echo $(ps -p $(cat $PIDFILE 2>/dev/null) ho pid 2>/dev/null)
```

Comment:
Ideally we want the PID file to log the PID of start-aws-kinesis-agent, so that babysitter would know if the Java process is abnormally killed or manually terminated by user. But I'm going to merge this PR for now so that Agent won't be restarted every minute.